src/common/flags/flags.ts

```diff
@@ -47,7 +47,7 @@ export const requiredDoceOrgFlag = OclifFlags.custom({
   summary: messages.getMessage('flags.targetDoceOrg.summary'),
   parse: async (input: string | undefined) => getOrgOrThrow(input),
   default: async () => getOrgOrThrow(),
-  defaultHelp: async () => (await getOrgOrThrow())?.getUsername(),
+  defaultHelp: async () => getDefaultDevopsCenterUsernameAlias(),
```

Comment:
defaultHelp is just for the flag help and should not throwing an exception when there is no default.